### ISSUE TITLE
Revise headings markup

### ISSUE THREAD

**Issue by mark-a-wilson** [bcgov/digital.gov.bc.ca#242]
Reference: [WCAG 1.3.1 - Info and Relationships](https://www.w3.org/TR/WCAG21/#info-and-relationships)

### Problems

- **Page titles** are currently marked up as `<p class="pageTitle">`. 
- **Titles of horizontal cards** are currently marked up as `<p class="cardTitleHorizontal">`.

Though this doesn't seem to be messing up keyboard navigation order with a screen reader, it _is_ something that gets flagged by an evaluation tool ([WAVE](https://wave.webaim.org)) as something that should be a heading.

### For Discussion
@singhinderdeep @thegentlemanphysicist - let's talk this through as we have a bit of a conundrum:

* The **site title** in the header ("Digital Government" beside the BC Gov mark) is currently marked up as `<h1>` (which seems to be correct according to the [Design System](https://developer.gov.bc.ca/Design-System/Header-Basic#html)).
* ↳ However, since any page should only have one `<h1>`, this means that the **page title** should be `<h2>` (will do some research on whether that's correct—it doesn't seem intuitive)
* For aesthetics, some of Inderdeep's wireframes use multiple `<h1>`s on a page (e.g. case studies). Hierarchical semantic markup needs to trump aesthetics.

### Acceptance Criteria
- [ ] When a **page title** is navigated to using a screen reader, it is read as a "heading, level ...."
- [ ] When a **horizontal card** is navigate to using a screen reader, its title is read as "heading, level ...."

### Definition of Done
- [ ] All tests (unit/functional/security/code cov.) pass
- [ ] Code review has been done
- [ ] Pipeline builds
- [ ] PO signs off on the build

### Examples
- [ ] **Digital Principles** page: `h2` level is skipped
- [ ] **Products & Services** page: headings missing for the most part

**Comment by mark-a-wilson** [comment]
@thegentlemanphysicist - discussed this one with Heather yesterday and she'd like us to tackle this sprint. Give me a call about it when we're picking up....

**Comment by thegentlemanphysicist** [comment]
- remove h1 from heading of website (digital gov )
- set title in banner to h1 object
- titles of card sections should be h2  or h3, never h1  
- banner simple title h1 (also resizing is buggy)
- 'for desiners' and 'for developers' will be h3's 

- set title of cards to h4 object (some are p, some are h2) ????

**Comment by mark-a-wilson** [comment]
Answers I'm going to try to find:

1. Is it OK for screen reader users if the markup skips a level of heading (e.g. `h1`, then `h3`, with no `h2`s on the page), or is this considered a problem (_Am I missing something on this page? There was no level 2._)?
2. Must **card titles** be marked up as headings to be understandable to screen reader users? Or are there other acceptable practices?

**Comment by thegentlemanphysicist** [comment]
In the second PR 

- card titles all h5

**Comment by thegentlemanphysicist** [comment]
This will be deployed at link https://digital-gov-frontend-pr-340-xdwiwd-dev.pathfinder.gov.bc.ca/case-studies/mining-study
The one part that was not completed is wrapping cards in ul and li.  This conflicted with the use of antd <Row> and <Col>
